upload/docker_images/validator/validator_harness.py

```diff
@@ -1,25 +1,22 @@
+import logging
 import os
-import sys
 import pathlib
 import subprocess
+import sys
 import time
 import urllib.parse
-import logging
 
 import boto3
-from urllib3.util import parse_url
 from tenacity import retry, stop_after_attempt, before_log, before_sleep_log, wait_exponential, TryAgain
+from urllib3.util import parse_url
 
-from upload.common.validation_event import ValidationEvent
-from upload.common.logging import get_logger
 from upload.common.upload_api_client import update_event
+from upload.common.validation_event import ValidationEvent
 
-
-logger = get_logger(f"CHECKSUMMER [{os.environ.get('AWS_BATCH_JOB_ID')}]")
+logger = logging.getLogger(f"CHECKSUMMER [{os.environ.get('AWS_BATCH_JOB_ID')}]")
```

Comment:
This is invoked in the docker image that runs in batch. What is the consequence of not calling configure_logging first?

Reply:
Talked with Andrey a bit and seems pretty reasonable and makes it super simple moving forward so did it. Thanks for the suggestion!